### DISCUSSION TITLE
Highlighting for the current cell?

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1093] [Q&A] [answered]
Is there a way to set the highlighting for the current cell?  I don't see a disp_* option that looks like it would allow users to add highlighting for current cell.   I make mistakes in Visidata because I think the cursor is somewhere else.

**Comment by saulpw** [comment]
It's a combination of `options.color_current_col` and `color_current_row`.

**Reply by frosencrantz** [reply]
It doesn't look like they combine.  It looks like color_current_row overrides the color_current_col.

I tried setting the current col to green, but it looks like the current cell is not shown in reverse green:
   https://asciinema.org/a/2LEqjL4H0DD6WM2WyNomqwXwF?t=10

**Reply by saulpw** [reply]
Attributes (underline/reverse/bold) will combine.  You can only have one color though (well, 2, fg and bg).  

Also (and I don't remember changing this) for me on the current version, column color overrides row color.

**Comment by saulpw** [comment]
Try this:

```
TableSheet.colorizers += [
    CellColorizer(3, 'bold red on black', lambda s,c,r,v: c is s.cursorCol and r is s.cursorRow)
]
```

**Reply by frosencrantz** [reply]
I've tried it for several days now, and that works well.  Thank you, @saulpw!

It is not possible to do only with options.  It seems like an option for styling the current cell might of interest.

**Reply by saulpw** [reply]
Okay, `options.color_current_cell` has been added to vdplus!

**Reply by frosencrantz** [reply]
Sounds great, @saulpw !  Though it seems like it would be useful in the main Visidata.

**Reply by saulpw** [reply]
I mean isn't that true of everything?  If anything I've been trying to move things out of main VisiData to make the core smaller and more tractable.  But sometimes I wonder if vdplus and all plugins should be in 'main' VisiData, at least the repo if installed and/or enabled by default.  The right scope for tools like this is not obvious.

**Comment by anjakefala** [comment, marked as answer]
@frosencrantz `options.color_current_cell` has now been moved to core VisiData!

**Reply by reagle** [reply]
Color specification is Xterm numbers/colors: https://www.ditig.com/256-colors-cheat-sheet

**Reply by frosencrantz** [reply]
Brilliant!  Thank you, @anjakefala!

**Reply by curiousponge** [reply]
How do we make us of your change on core VisiData, @anjakefala? Sorry for this noob question, but the latest release seems to be on January. Help would be appreciated.

**Reply by anjakefala** [reply]
Hi @curiousponge!

Until the next release, the best way to use the current development version is `pip3 git+https://github.com/saulpw/visidata.git`

**Reply by curiousponge** [reply]
Thank you very much @anjakefala! It works. I have learned a new thing.
